dockerfiles/splash/download-pyqt5.sh

```diff
@@ -2,10 +2,9 @@
 # XXX: these URLs needs to be replaced with sourceforge in future,
 # because riverbank tend to remove old releases.
 SIP="https://www.riverbankcomputing.com/static/Downloads/sip/4.19.19/sip-4.19.19.tar.gz"
-PYQT5="https://www.riverbankcomputing.com/static/Downloads/PyQt5/5.13.1/PyQt5_gpl-5.13.1.tar.gz"
-WEBENGINE="https://www.riverbankcomputing.com/static/Downloads/PyQtWebEngine/5.13.1/PyQtWebEngine_gpl-5.13.1.tar.gz"
+PYQT="https://www.riverbankcomputing.com/static/Downloads/PyQt${QT_MAJOR_VERSION}/${QT_FULL_VERSION}/PyQt${QT_MAJOR_VERSION}_gpl-${QT_FULL_VERSION}.tar.gz"
+WEBENGINE="https://www.riverbankcomputing.com/static/Downloads/PyQtWebEngine/${QT_FULL_VERSION}/PyQtWebEngine_gpl-${QT_FULL_VERSION}.tar.gz"
```

Comment:
I'm not sure QT and PyQT versions match; it seems PyQt 5.13.x supports Qt 5.13.x, but .x is updated separately. Also, https://github.com/scrapinghub/splash/pull/1038 shows that you can use a non-matching PyQT version.

The problem here is that it is not possible to update to PyQT 5.14 just by changing version number - package may be not available on the same URL, and they also changed the installation method (current scripts don't work).

Reply:
what if we will be building it ourselves?

Reply:
We are building PyQT and PyQtWebkit ourselves, using source packages; this PR adds an option to build qtwebkit ourselves.

Reply:
Riverbank is providing source tars in pypi now https://files.pythonhosted.org/packages/47/9f/60e630711fd1dd14ef3bd95c86c733c86b8c0853749c7a03691f681f13fd/PyQtWebEngine-5.14.0.tar.gz, https://files.pythonhosted.org/packages/4d/81/b9a66a28fb9a7bbeb60e266f06ebc4703e7e42b99e3609bf1b58ddd232b9/PyQt5-5.14.2.tar.gz and https://files.pythonhosted.org/packages/bc/dd/aa60ae73bc213466fd158e3d8b91d55ac00a248b14a98919bfe0e12c5d74/sip-5.2.0.tar.gz